common/autocomplete/awesomplete.js

```diff
@@ -0,0 +1,205 @@
+/* globals Awesomplete */
+
+// Dependencies
+
+let builder = Focus.component.builder;
+let types = Focus.component.types;
+let find = require('lodash/collection/find');
+
+/**
+ * Autocomplete component.
+ * Get a pickList as an input, then let the user type and suggests values from the picklist.
+ * Can force values in the input field to be taken from the pick list only.
+ * @type {Object}
+ */
+let Autocomplete = {
+    /**
+     * Component will mount.
+     * Check if the Awesomplete library is in the Window object.
+     */
+    componentWillMount() {
+        // Check if Awesomplete is set in Window
+        if (!window.Awesomplete) {
```

Comment:
Maybe add the awesomeplete as optional dependencies https://docs.npmjs.com/files/package.json#optionaldependencies